### PR TITLE
Initial design and development

### PR DESCRIPTION
**Pull request by bkmoovio**
Added tests and README Documentation

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
```
Comment (adamdecaf) [inline CODE_OF_CONDUCT.md]: I was just linking to this file in the ach project. Should we have one file that's linked to in all projects?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,169 @@
...
+ Google Group [moov-users](https://groups.google.com/forum/#!forum/moov-users)| The Moov users Google group is for contributors other people contributing to the Moov project. You can join them without a google account by sending an email to [moov-users+subscribe@googlegroups.com](mailto:moov-users+subscribe@googlegroups.com). After receiving the join-request message, you can simply reply to that to confirm the subscription.
+Twitter [@moov_io](https://twitter.com/moov_io)	| You can follow Moov.IO's Twitter feed to get updates on our project(s). You can also tweet us questions or just share blogs or stories.
+[GitHub Issue](https://github.com/moov-io) | If you are able to reproduce an problem please open a GitHub Issue under the specific project that caused the error.
+[moov-io slack](http://moov-io.slack.com/) | Join our slack channel to have an interactive discussion about the development of the project. [Request and invote to the slack channel](https://join.slack.com/t/moov-io/shared_invite/enQtNDE5NzIwNTYxODEwLTRkYTcyZDI5ZTlkZWRjMzlhMWVhMGZlOTZiOTk4MmM3MmRhZDY4OTJiMDVjOTE2MGEyNWYzYzY1MGMyMThiZjg)
```
Comment (adamdecaf) [inline README.md]: Can we fix the typo to "Request an invite"? 

I copy/pasted from the ach project and had to fix this elsewhere.

File: test/data/20180905A.x9 [hunk 3]
```diff
@@ -0,0 +1,23 @@
+0135T231380104121042882201810012313NCitadel           Wells Fargo        US
```
Comment (adamdecaf) [inline test/data/20180905A.x9]: FYI, directories named `testdata/` are skipped by the go tool.

```
$ go help test
..
The go tool will ignore a directory named "testdata", making it available
...
```

File: README.md (hunk 2 above)
Comment (bkmoovio) [reply README.md]: Yep

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (bkmoovio) [reply CODE_OF_CONDUCT.md]: Yeah I guess I though the reference to ACH in the URL would be misleading.  Is there a place in the repo, we can have the code of conduct without a reference to a specific library?

File: test/data/20180905A.x9 (hunk 3 above)
Comment (bkmoovio) [reply test/data/20180905A.x9]: That was the requirement for ACH.  Are you saying we don't want to have a testdata directory for x9? or are you saying we should eliminate testdata from both libraries.

File: README.md (hunk 2 above)
Comment (bkmoovio) [reply README.md]: Fixed!

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (adamdecaf) [reply CODE_OF_CONDUCT.md]: What about in the moov.io site? I'll offer to convert the markdown to html if we want. 

https://github.com/moov-io/moov-io

File: test/data/20180905A.x9 (hunk 3 above)
Comment (adamdecaf) [reply test/data/20180905A.x9]: Test files are totally fine with me. I was just mentioning the standard Go convention is to put them in a `testdata/` directory.

Comment (bkmoovio) [reply test/data/20180905A.x9]: I was following this, https://github.com/golang-standards/project-layout, which only had a test directory.  Is that not correct?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (bkmoovio) [reply CODE_OF_CONDUCT.md]: That should work.

File: test/data/20180905A.x9 (hunk 3 above)
Comment (adamdecaf) [reply test/data/20180905A.x9]: Oh. I didn't realize that project has `test/`. Either is fine with me. 

There's an open issue on that project https://github.com/golang-standards/project-layout/issues/3

**Pull request merged**